docs/section19/Rule 19-37.md

```diff
@@ -0,0 +1,35 @@
+# Section 19 - Rule 19-37 
+**Schema Version:** 0.1.3  
+**Mandatory Rule:** False    
+**Rule ID:** 19-3     
+**Rule Description:** Weather conditions used in sizing runs for the 99.6% heating design temperature and 1% dry-bulb and 1% wet-bulb cooling design temperatures are consistent with the site location.    
+**Rule Assertion:** Options are UNDETERMINED/NOT_APPLICABLE     
+**Appendix G Section:** G3.1.2.2.1      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each RulesetModelDescription
+
+**Applicability Checks:** None  
+
+**Function Calls:**  None
+
+## Check Applicability:
+- Set default applicability: `rule_applicability = True`  
+- Get the weather for the ruleset model description: `weather = RulesetModelDescription.weather`  
+- If the design days were incorrectly defined in terms of type in thr RPD then rule is NOT_APPLICABLE: `if not (weather.cooling_dry_bulb_temp_design_day_type == "COOLING_1_0" and weather.heating_dry_bulb_temp_design_day_type == "HEATING_99_6" and weather.cooling_wet_bulb_temp_design_day_type == "COOLING_1_0"):`  
```

Comment:
Note to self, add note that for the proposed this is only applicable if there are yet to be designed HVAC systems

Reply:
small comment- looks like 

cooling_dry_bulb_temp_design_day_type was renamed to CoolingDesignDayOptions
similarly,  heating_dry_bulb_temp_design_day_type was renamed to HeatingDesignDayOptions.

Reply:
@supriyagoel Can you please clarify what you mean? I think I have the correct labels, what you're referring to are the enumeration but I could be missing something so please let me know if I am. Thanks!